docs/language/operators/summarize.md

```diff
@@ -5,26 +5,29 @@
 ### Synopsis
 
 ```
-summarize [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>]] [by [<field>][:=<expr>] ...]
+summarize [<field>:=]<agg> [where <expr>][, [<field>:=]<agg> [where <expr>] ...] [by [<field>][:=<expr>] ...]
 ```
 ### Description
 
-The `summarize` operator consumes all of its input, applies an aggregate function
+The `summarize` operator consumes all of its input, applies an [aggregate function](../aggregates/README.md)
 to each input value optionally organized with the group-by keys specified after
-the `by` keyword and at the end of input, and produces one or more aggregations
+the `by` keyword, and at the end of input produces one or more aggregations
 for each unique set of group-by key values.
 
-Each aggregate function may be optionally followed by a `where` clause, which
-indicates a Boolean expression that indicates, for each input value,
-whether to deliver it to that aggregate.
+The keyword `summarize` can be treated as an [implied operator](../overview.md#26-implied-operators)
+and hence may be omitted before an aggregate function call.
+
+Each aggregate function may be optionally followed by a [`where`](where.md) clause, which
```

Comment:
Are folks ok with me linking to the `where` operator doc from here? I don't know if it's technically 100% the same thing since it's being used to affect the behavior of another operator rather than as a standalone operator. However, assuming they share the same code, I figure linking is the best way to answer the user's possible question of "What are the details on how this `where` clause is used?"

Reply:
I do think this is a bit confusing because this is a clause, not an operator. How about something like this?

```
Each aggregate function may be optionally followed by a `where` clause, which
applies a Boolean expression that indicates, for each input value,
whether to deliver it to that aggregate. (`where` clauses are analogous
to the [`where` operator](where.md).)
```